app/views/user_mailer/nsv_bsv_submission_rejected.html.erb

```diff
@@ -11,7 +11,11 @@
         </tr>
         <tr>
           <td class='email-sub-title'>
-            <p>Dear <%= @submission.name %>,</p>
+            <% if @submission.user_name %>
+              <p>Hi <%= @submission.user_name %>,</p>
+            <% else %>
+              <p>Hi there,</p>
+            <% end %>
             <p>Thank you for submitting <%= @submission.title %> by <%= @artist.name %> to us for evaluation. </p>
             <p>Unfortunately, we don’t have a selling opportunity for this work right now. However, we have uploaded it to <%= link_to 'My Collection', 'https://www.artsy.net/settings/my-collection', target: :_blank %> in Artsy to help you track demand.</p>
```

Comment:
**Suggestion:** Should we use https://www.artsy.net/my-collection as the URL to open the app if the user has the app?

Reply:
Yesssss. Thank youuu!